### ISSUE TITLE
Start doing research survey on LLMs and datasets

### ISSUE THREAD

**Issue by sonaalKant** [kaizen-ai/kaizenflow#750]
The task involves leveraging a Large Language Model (LLM) for sentiment analysis on financial news. The primary objective is to:

1. Find a Suitable Dataset:

Locate and acquire a publicly available dataset containing financial news articles (see if there are some on crypto). Create a spreadsheet with columns `[ dataset name, dataset source, size of dataset, Cost(Free/paid), label, any benchmark available or not, Contains crypto news ]`

2. Find Suitable LLMs:

Start with creating List of LLMs which are currently finetuned on financial data/news. Create a spreadsheet with columns `[Model Name, Model download link, Model trained/finetuned on datasets, evaluated on benchmark]`

For reference on dataset and publicly available models you can refer https://paperswithcode.com/ , https://huggingface.co/models and https://huggingface.co/datasets. 

FYI @gpsaggese @samarth9008 @jsmerix

**Comment by tkpratardan** [comment]
@sonaalKant 
What is the expected timeline?

**Comment by sonaalKant** [comment]
In the models sheet, `Evaluated on benchmark` meant what other test set they have evaluated on besides the test split.

**Comment by tkpratardan** [comment]
@sonaalKant 
Could you please review the docs [here](https://drive.google.com/drive/u/3/folders/1FKEkRZwshZhj_hSB-n0HaOkeneBiFnOY)?

Also, could you please provide your feedback on the following:
1. Am I headed in the right direction?
2. Any improvements and feedback on the work?

**Comment by sonaalKant** [comment]
This looks good. I will close this issue as it has covered preliminary research in models and datasets. I will file another issue about next steps.

**Comment by gpsaggese** [comment]
Good stuff.

**Comment by sonaalKant** [comment]
This is the first step towards PEOE. I think we have some preliminary findings /survey here https://drive.google.com/drive/u/3/folders/1FKEkRZwshZhj_hSB-n0HaOkeneBiFnOY
